### PR TITLE
Refactoring a method to not depend on the input but on the individual…

### PR DESCRIPTION
**Pull request by mikeSimonson**
… options

### CODE REVIEW

File: lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php [hunk 1]
```diff
@@ -117,7 +127,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $output->writeln(sprintf('Generated new migration class to "<info>%s</info>" from schema differences.', $path));
     }
 
-    private function buildCodeFromSql(Configuration $configuration, InputInterface $input, array $sql)
+    private function buildCodeFromSql(Configuration $configuration, array $sql, $formatted=false, $lineLength=0)
```
Comment (stof) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php]: ``0`` look like a weird default

Comment (mikeSimonson) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php]: false ?

Comment (stof) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php]: no. It needs to be an integer. But any value smaller than 27 will be broken (see the logic line 147)

Comment (mikeSimonson) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php]: 80 then.

Comment (stof) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php]: yeah, or 120 to be consistent with the default value of the CLI option itself :)

Comment (mikeSimonson) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php]: ok thanks

Comment (mikeSimonson) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php]: done

**Pull request merged**